tests/onnx/test_onnx_model_export.py

```diff
@@ -22,6 +25,10 @@
 from mlflow.utils.model_utils import _get_flavor_configuration
 from mlflow.tracking._model_registry import DEFAULT_AWAIT_MAX_SLEEP_SECONDS
 
+
+TEST_DIR = "tests"
+TEST_ONNX_RESOURCES_DIR = os.path.join(TEST_DIR, "resources", "onnx")
```

Comment:
Rather than constructing multi-input / list output models in TensorFlow & scikit-learn, which requires the use of `onnxmltools` in order to export models in ONNX format, we instead reload pretrained / serialized models from the resources folder.

In order to preserve a true end-to-end example from training through inference, I've opted to include a `torch.onnx` workflow here in place of the preexisting  tf/keras workflow.